index.html

```diff
@@ -289,6 +289,36 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+
+      <div id="contact-form">
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h2>
+        <form action="https://formspree.io/f/mwkagzpj" method="POST">
+          <input 
+            type="text" 
+            id="fullname" 
+            name="fullname"
+            placeholder="Full Name"
+            maxlength="30"
+            required
+          />
+          <input 
+            type="email" 
+            id="email" 
+            name="email"
+            placeholder="Email address"
+            required
+          />
```

Comment:
- [ ] Kindly add `maxlength="30"` as an attribute to the `email` input field. This will help to control the number of letters that can be typed in as an email.